src/world/Objects/Units/Unit.Legacy.cpp

```diff
@@ -6944,11 +6939,10 @@ DamageInfo Unit::Strike(Unit* pVictim, WeaponDamageType weaponType, SpellInfo co
 
             if (this->isPlayer() && this->getClass() == WARRIOR)
             {
-                static_cast<Player*>(this)->AddComboPoints(pVictim->getGuid(), 1);
-                static_cast<Player*>(this)->UpdateComboPoints();
+                static_cast<Player*>(this)->addComboPoints(pVictim->getGuid(), 1);
```

Comment:
this is a Unit pointer, since Player inherits from Unit this should be a dynmic_cast instead of a static_cast